src/Apps/CollectorProfile/Routes/Saves2/Components/ArtworkListForm/ArtworkListForm.tsx

```diff
@@ -14,6 +14,7 @@ export const MAX_NAME_LENGTH = 40
 export const validationSchema = Yup.object().shape({
   name: Yup.string()
     .required(i18n.t("collectorSaves.artworkListForm.fields.name.required"))
+    .matches(/\S+/, { message: "name cannot be empty" })
```

Comment:
Keeping this new validation simple: at least one non-whitespace character is required, anywhere in the name.